src/workerd/api/streams/readable.h

```diff
@@ -365,10 +365,13 @@ class ReadableStream: public jsg::Object {
   void initEofResolverPair(jsg::Lock& js);
   // By default the eofResolverPair is not initialised, calling this method will initialise it so
   // that the signalling is active.
+  bool getPipeToCalled() { return pipeToCalled; };
+  // Determines whether `pipeTo` was called on this ReadableStream.
```

Comment:
The comment is a bit tautological to the message name. It would likely be better to explain a bit why this is important and how it is used, especially given that there's nothing in the stream standard that says this is needed.